core/src/main/scala/za/co/absa/spline/harvester/dispatcher/CompositeLineageDispatcher.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.harvester.dispatcher
+
+import za.co.absa.commons.config.ConfigurationImplicits._
+import za.co.absa.spline.harvester.conf.HierarchicalObjectFactory
+import za.co.absa.spline.harvester.dispatcher.CompositeLineageDispatcher._
+import za.co.absa.spline.producer.model.v1_1.{ExecutionEvent, ExecutionPlan}
+
+class CompositeLineageDispatcher(delegatees: LineageDispatcher*) extends LineageDispatcher {
+
+  def this(objectFactory: HierarchicalObjectFactory) = this(getDispatchers(objectFactory): _*)
+
+  override def send(plan: ExecutionPlan): Unit = delegate {
+    _.send(plan)
+  }
+
+  override def send(event: ExecutionEvent): Unit = delegate {
+    _.send(event)
+  }
+
+  private def delegate(call: LineageDispatcher => Unit): Unit = {
+    delegatees.foreach(call)
```

Comment:
Shouldn't there be some exception handling, so exception in one dispatcher doesn't stop the other?

Reply:
added a property `failOnErrors` which if set to `true` re-throws exceptions and fails fast,
or otherwise suppresses the errors. logging them as WARN and proceeds.